### PR TITLE
add slider, text utilities and toast components

### PR DESCRIPTION
**Pull request by abhisheksunil2201**
added slider, text utilities and toast components

### CODE REVIEW

File: components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.slider[type="range"] {
+  -webkit-appearance: none;
+}
+input.slider[type="range"]::-webkit-slider-runnable-track {
+  background: var(--cta-color);
+  border-radius: 5rem;
+  -webkit-appearance: none;
+  margin: 0 1rem;
+}
+input.slider[type="range"]::-webkit-slider-thumb {
```
Comment (Sharmaa-Ansh) [inline components/Slider/slider.css:10]: Can use rem instead of px

File: components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  background-color: var(--success-color);
+  color: white;
+}
+.toast-error {
+  color: white;
+  background-color: var(--danger-color);
+}
+.toast-warning {
+  background-color: var(--warning-color);
+  color: white;
```
Comment (Sharmaa-Ansh) [inline components/Toast/toast.css:24]: white comes up often might as well turn it into a variable

**Review by Sharmaa-Ansh — APPROVED** [review]

File: components/Slider/slider.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          </li>
+          <li class="nav-link-item active">
+            <a href="/components/Slider/slider.html">Slider</a>
+          </li>
+          <li class="nav-link-item">
+            <a href="/components/Toast/toast.html">Toast</a>
+          </li>
+        </ul>
+        <div style="height: 1.5rem"></div>
+      </div>
```
Comment (ddivyansh18) [inline components/Slider/slider.html:72]: Avoid inline CSS styling

File: components/Slider/slider.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      <!-- Ham menu -->
+      <div class="ham-nav">
+        <a href="/index.html"><h1 class="ham-logo">REDUCTO UI</h1></a>
+        <div>
+          <img
+            id="ham-menu"
+            class="ham-menu"
+            src="/assets/ham-menu.svg"
+            alt=""
+          />
```
Comment (ddivyansh18) [inline components/Slider/slider.html:82]: Fill alt tag of image appropriately

File: components/Text Utilities/text-utilities.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+          </li>
+          <li class="nav-link-item">
+            <a href="/components/Slider/slider.html">Slider</a>
+          </li>
+          <li class="nav-link-item">
+            <a href="/components/Toast/toast.html">Toast</a>
+          </li>
+        </ul>
+        <div style="height: 1.5rem"></div>
+      </div>
```
Comment (ddivyansh18) [inline components/Text Utilities/text-utilities.html:72]: Avoid inline CSS styling

File: components/Text Utilities/text-utilities.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+      <!-- Ham menu -->
+      <div class="ham-nav">
+        <a href="/index.html"><h1 class="ham-logo">REDUCTO UI</h1></a>
+        <div>
+          <img
+            id="ham-menu"
+            class="ham-menu"
+            src="/assets/ham-menu.svg"
+            alt=""
+          />
```
Comment (ddivyansh18) [inline components/Text Utilities/text-utilities.html:82]: Fill alt tag of image appropriately

File: components/Toast/toast.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+          </li>
+          <li class="nav-link-item">
+            <a href="/components/Slider/slider.html">Slider</a>
+          </li>
+          <li class="nav-link-item active">
+            <a href="/components/Toast/toast.html">Toast</a>
+          </li>
+        </ul>
+        <div style="height: 1.5rem"></div>
+      </div>
```
Comment (ddivyansh18) [inline components/Toast/toast.html:72]: Avoid inline CSS styling

**Review by ddivyansh18 — CHANGES_REQUESTED** [review]

**Pull request merged**